src/languages/sqlite/sqlite.formatter.ts

```diff
@@ -73,7 +73,10 @@ export default class SqliteFormatter extends Formatter {
       reservedDependentClauses: ['WHEN', 'ELSE'],
       reservedKeywords: keywords,
       reservedFunctionNames: functions,
-      stringTypes: [{ quote: "''", prefixes: ['X'] }],
+      stringTypes: [
+        { quote: "''", prefixes: ['X'] },
+        { quote: '""', prefixes: ['X'] },
```

Comment:
Should not add `""` strings to SQLite - that will conflict with `""`-quoted identifiers.

Looks like SQLite has some strange rules where it sometimes treats double-quoted text as an identifier and sometimes as a string. And vice versa with single quotes. See: https://www.sqlite.org/lang_keywords.html